crates/runtime/tests/endpoint_auth/flight.rs

```diff
@@ -79,11 +79,17 @@ async fn test_flight_auth() -> Result<(), anyhow::Error> {
         .expect("to connect to flight endpoint");
     let client = FlightServiceClient::new(channel);
 
-    let result =
-        flightrepl::get_records(client.clone(), "SELECT 1", Some(&"valid".to_string())).await;
+    let result = flightrepl::get_records(
+        client.clone(),
+        "SELECT 1",
+        Some(&"valid".to_string()),
+        "test_flight_auth/1.0",
+    )
+    .await;
     assert!(result.is_ok());
 
-    let Err(e) = flightrepl::get_records(client, "SELECT 1", None).await else {
+    let Err(e) = flightrepl::get_records(client, "SELECT 1", None, "test_flight_auth/1.0").await
```

Comment:
This version should use the Cargo.toml version from the `env!("CARGO_PKG_VERSION")` macro